docs/howto/connection/ssh.rst

```diff
@@ -66,4 +66,4 @@ Extra (optional)
 
     .. code-block:: bash
 
-        ssh://user:pass@localhost:22?timeout=10&compress=false&no_host_key_check=false&allow_host_key_change=true&key_file=%2Fhome%2Fairflow%2F.ssh%2Fid_rsa
+        export AIRFLOW_CONN_MAIN_SERVER='ssh://user:pass@localhost:22?timeout=10&compress=false&no_host_key_check=false&allow_host_key_change=true&key_file=%2Fhome%2Fairflow%2F.ssh%2Fid_rsa'
```

Comment:
SSH Connection ID does not have a default connection ID, we should probably not set this.

Reply:
This is an example of how to set up any connection, not just the default connection. Configuration from variables is read every time you try to get the connection. The purpose of this PR is to improve the examples to show the code more close to real use, but not necessarily to configure the default connection. I think that this change is also helpful in this case, because it allows you to understand how to set the connection using environment variables. I often configure the connection using variable environments and I would like to be able to look only at one place for documentation and have a ready example.

Reply:
The default connection is configured elsewhere because this is the most common use.

Reply:
Check comment here - https://github.com/apache/airflow/pull/5973/files#r319935691

Reply:
Yeah. @kaxil -> this is all about the environment connections, so this is pretty fined here.